Makefile

```diff
@@ -394,6 +395,13 @@ openstack_crds: ## installs all openstack CRDs. Useful for infrastructure dev
 	for X in $$(file ${OUT}/openstack_crds/* | grep gzip | cut -f 1 -d ':'); do tar xvf $$X -C ${OUT}/openstack_crds/; done
 	for X in $$(grep -l CustomResourceDefinition ${OUT}/openstack_crds/manifests/*); do oc apply -f $$X; done
 
+.PHONY: openstack_storage_crds
```

Comment:
Since this is basically the same logic we already have in the openstack_crds target could we just reuse that same target by setting a different ENV for OPENSTACK_BUNDLE_IMG first? Reason I say this is we plan on splitting out more bundles soon so we'll likely need to do the same thing again.

Reply:
Thanks for the suggestion, I've updated the PR, reusing the `openstack_crds` target.